app/controllers/registrations_controller.rb

```diff
@@ -0,0 +1,25 @@
+class RegistrationsController < Devise::RegistrationsController
+  DEFAULT_REGISTRATION_PARAMS = { active: false, self_registered: true }.freeze
+  PERMITTED_PARAMS = %i[email full_name cellphone cpf address].freeze
+
+  before_action :configure_permitted_parameters
+  after_action :send_user_self_registration_notification_email, only: %i[create]
+
+  respond_to :json
+
+  private
+
+  def build_resource(params = {})
+    super(params.merge(DEFAULT_REGISTRATION_PARAMS))
+  end
+
+  def configure_permitted_parameters
+    devise_parameter_sanitizer.permit(:sign_up, keys: PERMITTED_PARAMS)
+  end
+
+  def send_user_self_registration_notification_email
+    return unless resource.persisted?
+
+    AdminMailer.user_self_registration_notification(admins: User.admins.to_a, user: resource).deliver_later
```

Comment:
Here you don't need to call `.to_a` on `User.admins`

Reply:
Unfortunately, for this case it is necessary to specify the `to_a` option for the admins. If this option is omitted, ActiveJob raises an exception:

```console
ActiveJob::SerializationError:
       Unsupported argument type: ActiveRecord::Relation
```

Reply:
What do you think about sending only the `emails` themselves? It seems to me you don't need the entire admin and by doing that you'll simplify the serialization and make the mailer know only what it needs to know to complete its job.